messenger.go

```diff
@@ -248,6 +277,7 @@ func NewMessenger(
 		shutdownTasks: []func() error{
 			persistence.Close,
 			adapter.transport.Reset,
+			func() error { datasync.Node.Stop(); return nil },
```

Comment:
Should we discard an error in this case?

Reply:
the function does not return any error I believe